utils/validate-data-source.js

```diff
@@ -1,13 +1,31 @@
 const appRoot = require('app-root-path');
 
-const { validateFilePath } = appRoot.require('api/v1/db/json/fs-operations');
-
-const dbPath = 'tests/unit/mock-data.json';
+const { validateJsonDb } = appRoot.require('api/v1/db/json/fs-operations');
+const { validateOracleDb } = appRoot.require('api/v1/db/oracledb/connection');
```

Comment:
When trying to run the application, I get this error:
```
internal/modules/cjs/loader.js:611
    throw err;
    ^

Error: Cannot find module '/Users/ianbrown/github/express-api-skeleton/api/v1/db/oracledb/connection'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:609:15)
    at Function.Module._load (internal/modules/cjs/loader.js:535:25)
    at Module.require (internal/modules/cjs/loader.js:663:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.require (/Users/ianbrown/github/express-api-skeleton/node_modules/app-root-path/lib/app-root-path.js:14:11)
    at Object.<anonymous> (/Users/ianbrown/github/express-api-skeleton/utils/validate-data-source.js:4:38)
    at Module._compile (internal/modules/cjs/loader.js:734:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:745:10)
    at Module.load (internal/modules/cjs/loader.js:626:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:566:12)
```
Since the skeleton comes with `api/v1/db/oracledb/connection-example.js` because it doesn't actually use an oracle db connection. I'm wondering what we should do here. I don't think apis should need this file to run since many won't use an oracle db connection. I'm starting to wonder if we should require the specific validation method in `app.js`, call the method there, and remove `utils/validate-data-source`. The method here is starting to look like a large wrapper that isn't adding much.

For example, if we want to validate a JSON database, we could simply do this in `app.js`:
```js
const { validateJsonDb } = appRoot.require('api/v1/db/json/fs-operations');
...
validateJsonDb();
```
and get the same functionality. What are your thoughts on this?

Reply:
Also, after fixing the require problem, there's another issue since `validateDataSource()` is called without any arguments in `app.js`